### PR TITLE
docs(build): add a plugin to display tag

### PR DESCRIPTION
**Pull request by tolking**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

```md
^(beta)
^(deprecated)
^(a11y)
^(2.3.0)
^(other)
```

### CODE REVIEW

**Review by chenxch — APPROVED** [review]

File: docs/.vitepress/plugins/tag.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+   */
+  md.renderer.rules.tag = (tokens, idx) => {
+    const token = tokens[idx]
+    const value = token.content
+    /**
+     * Add styles for some special tags
+     * vitepress/styles/content/tag-mark-content.scss
+     */
+    const tagClass = ['beta', 'deprecated', 'a11y'].includes(value) ? value : ''
+    return `<span class="vp-tag ${tagClass}">${value}</span>`
```
Comment (jw-foss) [inline docs/.vitepress/plugins/tag.ts:15]: **Question:** can we render them into Vue components?

**Review by jw-foss — APPROVED** [review]
Overall this LGTM, thanks 👍

File: docs/.vitepress/vitepress/styles/content/tag-content.scss [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  }
+  &.a11y {
+    --vp-tag-color: #6222c2;
+    --vp-tag-border-color: #9065db;
+  }
+}
+
+.dark .vp-tag {
+  &.a11y {
+    --vp-tag-color: #9065db;
```
[the remark this answers is not in the document]
Comment (jw-foss) [reply docs/.vitepress/vitepress/styles/content/tag-content.scss:30]: We could make this a follow up, I will merge this in advance.

[the remark this answers is not in the document]
Comment (jw-foss) [reply docs/.vitepress/vitepress/styles/content/tag-content.scss]: Let's do a followup pull request on this. Including documentation and code refinement.

**Pull request merged**